benchmarks/benchmark_dht.py

```diff
@@ -1,35 +1,120 @@
 import argparse
+import asyncio
 import random
 import time
+import uuid
 
+import numpy as np
 from tqdm import trange
 
 import hivemind
-from hivemind.moe.server import declare_experts, get_experts
-from hivemind.utils.limits import increase_file_limit
 from hivemind.utils.logging import get_logger, use_hivemind_log_handler
 
 use_hivemind_log_handler("in_root_logger")
 logger = get_logger(__name__)
 
 
-def random_endpoint() -> hivemind.Endpoint:
-    return (
-        f"{random.randint(0, 256)}.{random.randint(0, 256)}.{random.randint(0, 256)}."
-        f"{random.randint(0, 256)}:{random.randint(0, 65535)}"
+async def store_task(peer, key, value, expiration):
+    subkey = uuid.uuid4().hex
+
+    store_ok = await peer.store(
+        key, subkey=subkey, value=value, expiration_time=hivemind.get_dht_time() + expiration, return_future=True
+    )
+
+    return store_ok, subkey, value
+
+
+async def store_and_get_task(
+    peers: list,
+    total_num_rounds: int,
+    num_store_peers: int,
+    num_get_peers: int,
+    wait_after_iteration: float,
+    delay: float,
+    expiration: float,
+    latest: bool,
+    failure_rate: float,
+):
+    total_stores = total_gets = 0
+    successful_stores = []
+    successful_gets = []
+    store_times = []
+    get_times = []
+
+    for _ in range(total_num_rounds):
+        key = uuid.uuid4().hex
+        value = uuid.uuid4().hex
+
+        store_start = time.perf_counter()
+        store_peers = random.sample(peers, min(num_store_peers, len(peers)))
+        store_tasks = [store_task(peer, key, value, expiration) for peer in store_peers]
```

Comment:
```suggestion
        subkeys = [uuid.uuid4().hex for peer in store_peers]
        store_tasks = [peer.store(
            peer, key, subkey=subkey, value=value, get_dht_time() + expiration, return_future=True)
            for peer, subkey in zip(store_peers, subkeys)]
```

To the best of my knowledge, this coro is only used once. I would recommend either of:
- inlining it: see suggestion above
- or formatting it: add docstring and type hints